app/src/main/java/com/android/consumerfinancehalan/data/repo/SearchRepoImpl.kt

```diff
@@ -0,0 +1,4 @@
+package com.android.consumerfinancehalan.data.repo
+
+class SearchRepoImpl {
+}
```

Comment:
why do we have to create class for SearchRepoImpl in our "architecture Branch"

Reply:
Because we can't merge an empty package